public/app/features/dashboard/components/PanelEditor/getPanelFrameOptions.tsx

```diff
@@ -49,6 +50,7 @@ export function getPanelFrameCategory(props: OptionPaneRenderProps): OptionsPane
         render: function renderTitle() {
           return (
             <Input
+              data-testid={selectors.components.PanelEditor.OptionsPane.fieldInput('Title')}
```

Comment:
Looking at these selectors I wonder why aren't we using [Cypress testing library](https://testing-library.com/docs/cypress-testing-library/intro) as it has the same idea as React Testing Library and would allow us to reuse the existing ways to query elements?

Reply:
Sorry I missed this comments. 

There are many other factors affecting the choice of e2e testing framework for plugin e2e testing. You can find more information in [this](https://docs.google.com/document/d/1vLId8Gv8OQWqZZbgHfpy91ljARzs5ihfN7KxEQdN07U/edit#heading=h.5sybau7waq2q) design doc. TLDR; Playwright was chosen because it's more extensible than Cypress. 

Anyway, Cypress also has some kind of actionability checks so I think we would have faced the same issues there. .